### PR TITLE
OKTA-689678 - Hook doc set updates

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added two single-source global components for single sourcing in the hook docs; see-alsos updated with all hook guides; minor procedure updates in the Registration Hook to remove ambiguity
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-689678 ](https://oktainc.atlassian.net/browse/OKTA-689678 )

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9be0317b-1109-4b7b-a836-3f87e8e41ed2) | 74  | 69  | 100  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-implementation/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-implementation%2Fmain%2Findex.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4e5bd4a4-9633-4a49-aa85-8e2a927d29c7) | 74  | 81  | 77  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fpassword-import-inline-hook%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e7ee8a66-25c6-4590-ada1-a5f711474f90) | 80  | 70  | 76  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fregistration-inline-hook%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6e0d91e1-8ebc-44f8-83ac-daaff770b13f) | 31  | 84  | 85  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/saml-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsaml-inline-hook%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/10e273dd-4e41-4a26-b5a4-6277686623c7) | 96  | 73  | 77  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9c2760a4-1bbe-40cb-841d-d9c50b3a0e85) | 92  | 72  | 78  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/41647bd9c0ed30de1762e25345f6338225eca70d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftoken-inline-hook%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9421fd07-f3c8-4350-8e6d-bf5a38384d21) | 80  | 80  | 67  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b74b7fa0-9647-44e9-b8aa-a67f531f326d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-theme-prose/global-components/HookBasicAuthValuesNote.vue [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<template>
+  <blockquote>
+    <p>
+      <strong>Note</strong>: The sample application uses Basic Authentication. Ensure you add the authentication credentials to
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/global-components/HookBasicAuthValuesNote.vue:4]:
```suggestion
      <strong>Note</strong>: The sample application uses Basic Authentication. Ensure that you add the authentication credentials to 
```

File: packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md [hunk 2]
```diff
@@ -370,8 +378,8 @@ To associate the registration inline hook with a profile enrollment policy and a
 
 1. In the Admin Console, go to **Security > Profile Enrollment**.
 1. Click **Add Profile Enrollment Policy**.
-1. Give your policy a name (in this example, use "Inline Hook"), and then click **Save**.
-1. From the list of policies, find **Inline Hook**, and then click the pencil icon.
+1. Give your policy a name (in this example, use "Progressive Inline Hook"), and then click **Save**.
+1. Find your inline hook, "Progressive Inline Hook", from the list of Enrollment Policies, and then click the pencil icon.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md:382]:
```suggestion
1. Find your inline hook, "Progressive Inline Hook", from the list of enrollment policies, and then click the pencil icon.
```

File: packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md [hunk 3]
```diff
@@ -527,8 +539,8 @@ To associate the registration inline hook with a profile enrollment policy and a
 
 1. In the Admin Console, go to **Security > Profile Enrollment**.
 1. Click **Add Profile Enrollment Policy**.
-1. Give your policy a name (in this example, use "Inline Hook"), and then click **Save**.
-1. From the list of Enrollment Policies, find **Inline Hook** and click the pencil icon.
+1. Give your policy a name (in this example, use "SSR + PP Inline Hook"), and then click **Save**.
+1. Find your inline hook, "SSR + PP Inline Hook", from the list of Enrollment Policies, and then click the pencil icon.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md:543]:
```suggestion
1. Find your inline hook, "SSR + PP Inline Hook", from the list of enrollment policies, and then click the pencil icon.
```

**Review by susanharper-okta — COMMENTED** [review]
couple comments

File: packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md:543]: And fixed a third as well.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4c7189fa-c9c7-4fd2-8ee8-fded2046b76a) | 74  | 69  | 100  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-implementation/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-implementation%2Fmain%2Findex.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2b69053d-2d07-49ad-b1b9-2213c2dd26ff) | 74  | 81  | 77  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/password-import-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fpassword-import-inline-hook%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f2f8522f-b104-4d63-a964-4d5fa9da257f) | 80  | 70  | 76  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fregistration-inline-hook%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c93b98fa-48eb-46ad-baa4-e8352b6f4476) | 31  | 84  | 85  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/saml-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsaml-inline-hook%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5add2b02-159b-433a-92c2-2417493fd7a4) | 96  | 73  | 77  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftelephony-inline-hook%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dcc15789-a240-4c7a-9686-589999e3e91f) | 92  | 72  | 78  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md](https://github.com/okta/okta-developer-docs/blob/ecf5e7e2fb1b8b4301ff3c9157740ade0bae67e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Ftoken-inline-hook%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a120d5c9-abb8-4e70-83a9-847137edb689) | 80  | 80  | 67  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0c94306b-659b-4cc5-a69c-d9a0aa5cd8ca?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**